recipes/r-ggsignif/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set version = '0.4.0' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-ggsignif
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: ggsignif_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/ggsignif_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/ggsignif/ggsignif_{{ version }}.tar.gz
+  sha256: c7a1d3155cd848d008762a7aa15453e6f09fa7f719c60899995d8d754f43c5e6
+
+build:
+  number: 0
+  skip: true  # [win32]
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - r-ggplot2 >=2.0.0
+
+  run:
+    - r-base
+    - r-ggplot2 >=2.0.0
+
+test:
+  commands:
+    - $R -e "library('ggsignif')"  # [not win]
+    - "\"%R%\" -e \"library('ggsignif')\""  # [win]
+
+about:
+  home: https://github.com/const-ae/ggsignif
+  license: GPL-3
+  summary: Enrich your 'ggplots' with group-wise comparisons. This package provides an easy way
+    to indicate if two groups are significantly different. Commonly this is shown by
+    a bracket on top connecting the groups of interest which itself is annotated with
+    the level of significance (NS, *, **, ***). The package provides a single layer
+    (geom_signif()) that takes the groups for comparison and the test (t.test(), wilcox.text()
+    etc.) as arguments and adds the annotation to the plot.
+  license_file: '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'   # [unix]
+  license_file: '{{ environ["PREFIX"] }}\R\share\licenses\GPL-2'       # [win]
```

Comment:
Could you please change this to GPL-3? Did you add these lines manually or with the helper script?

Reply:
I might have done it manually. Can't remember...

Ok, I've rerun the helper script locally, it gave the correct version. So that was me, for sure :yum: